creator-node/src/dbManager.js

```diff
@@ -433,6 +433,51 @@ class DBManager {
       (indexCreation) => indexCreation.relname === 'Files_storagePath_idx'
     )?.length
   }
+
+  static async getNonDirLegacyStoragePathsAndCids(cursor, batchSize) {
+    const queryResult = await models.File.findAll({
+      attributes: ['storagePath', 'multihash'],
+      where: {
+        multihash: { [sequelize.Op.gte]: cursor },
```

Comment:
need to order by multihash right?

Reply:
following up from our offline sync about this - ordering will be technically correct and still run in seconds so not too much resource impact 👍 added